app/serializers/rest/announcement_serializer.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class REST::AnnouncementSerializer < ActiveModel::Serializer
+  attributes :id, :content, :starts_at, :ends_at
+
+  has_many :mentions
+  has_many :tags, serializer: REST::StatusSerializer::TagSerializer
+  has_many :emojis, serializer: REST::CustomEmojiSerializer
+
+  def id
+    object.id.to_s
+  end
+
+  def content
+    Formatter.instance.linkify(object.text)
+  end
+
+  class AccountSerializer < ActiveModel::Serializer
```

Comment:
Is this serializer used for `mentions`? If so, I'm not sure I understand why/how. `REST::StatusSerializer`'s is called `MentionSerializer` so I'm not sure what's different here…?

Reply:
Status model has a Mention association. Here, we use Account models directly.